model/system/metrics.yaml

```diff
@@ -32,6 +32,20 @@ groups:
     unit: "{cpu}"
     attributes: []
 
+  - id: metric.system.cpu.utilization
+    type: metric
+    metric_name: system.cpu.utilization
+    stability: development
+    brief: Fraction of total CPU time spent in active (non-idle) modes.
```

Comment:
Should we mention that it's the summary for all of the cores of the system/host?

Reply:
Definitely, added in https://github.com/open-telemetry/semantic-conventions/pull/2088/commits/fc50a970aef5e44cef24f2fb82f26a69942ff84f